src/Uno.UI.Runtime.WebAssembly/Xaml/Media/FontFamilyHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Media;
+
+namespace Uno.UI.Xaml.Media;
+
+/// <summary>
+/// WebAssembly specific <see cref="FontFamily"/> helper
+/// </summary>
+public partial class FontFamilyHelper
+{
+	/// <summary>
+	/// Pre-loads a font to minimize loading time and prevent potential text re-layouts.
+	/// </summary>
+	/// <returns>True is the font loaded successfuly, otherwise false.</returns>
+	public static Task<bool> PreloadAsync(FontFamily family)
+		=> FontFamily.PreloadAsync(family);
+
+	/// <summary>
+	/// Pre-loads a font to minimize loading time and prevent potential text re-layouts.
+	/// </summary>
+	/// <returns>True is the font loaded successfuly, otherwise false.</returns>
+	public static Task<bool> PreloadAsync(string familyName)
+		=> PreloadAsync(new FontFamily(familyName));
```

Comment:
Should the two methods be `async` and the body be `await`ed?

Reply:
Those methods are already invoking task-returning methods, there's no need to make those async.